lib/data/wrapper.js

```diff
@@ -21,6 +22,14 @@ if (config.backends.data === 'mem') {
 } else if (config.backends.data === 'multiple') {
     client = multipleBackendGateway;
     implName = 'multipleBackends';
+} else if (config.backends.data === 'cdmi') {
```

Comment:
There is no plan to integrate the CDMI back-end into the location constraint feature for the first version but it will be the case for the second one.